### PR TITLE
Improve naming for per-phase component metric streams

### LINKED ISSUE #880: Name single phase and 3-phase metrics consistently
### What's needed?

The name for the metrics streamers are not consistent in term of single phase or 3-phase values that they stream.

```python
microgrid.grid().power.new_receiver() # single phase values

microgrid.grid().current.new_receiver() # 3-phase tuple

microgrid.voltage().new_receiver() # 3-phase tuple
 
microgrid.frequency.new_receiver() # single phase values
```

The SDK will be adding more 3-phase metrics in the near future so it'll be helpful to distinguish the type of values provided by each metric in the name.

### Proposed solution

A first idea is to append a suffix to the 3-phase metrics (e.g. `_per_phase`). This still needs to be discussed and agreed on.

```python
microgrid.grid().power.new_receiver() # single phase values

microgrid.grid().current_per_phase.new_receiver() # 3-phase tuple

microgrid.voltage_per_phase().new_receiver() # 3-phase tuple
 
microgrid.frequency.new_receiver() # single phase values
```

### Use cases

_No response_

### Alternatives and workarounds

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by shsms**
Closes #880

### CODE REVIEW

File: src/frequenz/sdk/microgrid/_data_pipeline.py [hunk 1]
```diff
@@ -139,7 +139,7 @@ def frequency(self) -> GridFrequency:
 
         return self._frequency_instance
 
-    def voltage(self) -> VoltageStreamer:
+    def voltage_per_phase(self) -> VoltageStreamer:
         """Return the 3-phase voltage measuring point."""
```
Comment (daniel-zullo-frequenz) [inline src/frequenz/sdk/microgrid/_data_pipeline.py]: Maybe `Return the per-phase voltage measuring point.`?

File: src/frequenz/sdk/microgrid/_data_pipeline.py [hunk 2]
```diff
@@ -506,9 +506,9 @@ def frequency() -> GridFrequency:
     return _get().frequency()
 
 
-def voltage() -> VoltageStreamer:
+def voltage_per_phase() -> VoltageStreamer:
     """Return the 3-phase voltage measuring point."""
```
Comment (daniel-zullo-frequenz) [inline src/frequenz/sdk/microgrid/_data_pipeline.py]: what about `per-phase` instead of `3-phase`?

**Review by daniel-zullo-frequenz — COMMENTED** [review]
Just a few minor suggestions about docstrings to check for. LGTM otherwise

File: src/frequenz/sdk/microgrid/_data_pipeline.py (hunk 1 above)
Comment (shsms) [reply src/frequenz/sdk/microgrid/_data_pipeline.py]: true, fixed.

File: src/frequenz/sdk/microgrid/_data_pipeline.py (hunk 2 above)
Comment (shsms) [reply src/frequenz/sdk/microgrid/_data_pipeline.py]: fixed.

**Comment by daniel-zullo-frequenz** [comment]
The docs generation failed due to these warnings:

```
WARNING -  mkdocs_autorefs: reference/frequenz/sdk/timeseries/ev_charger_pool/index.md: Could not find cross-reference target '[frequenz.sdk.timeseries.ev_charger_pool.EVChargerPool.current]'
WARNING -  mkdocs_autorefs: reference/frequenz/sdk/timeseries/formula_engine/index.md: Could not find cross-reference target '[frequenz.sdk.timeseries.grid.Grid.current]'
WARNING -  mkdocs_autorefs: reference/frequenz/sdk/timeseries/formula_engine/index.md: Could not find cross-reference target '[frequenz.sdk.timeseries.ev_charger_pool.EVChargerPool.current]'
WARNING -  mkdocs_autorefs: user-guide/formula-engine.md: Could not find cross-reference target '[frequenz.sdk.timeseries.grid.Grid.current]'
WARNING -  mkdocs_autorefs: user-guide/formula-engine.md: Could not find cross-reference target '[frequenz.sdk.timeseries.ev_charger_pool.EVChargerPool.current]'
```

**Comment by shsms** [comment]
> The docs generation failed due to these warnings:

is fixed now.

**Review by daniel-zullo-frequenz — APPROVED** [review]

**Pull request merged**